Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.12-alpine
+
+RUN apk add --no-cache git
```

Comment:
Consider installing `make` here as well by changing this to: 
```
RUN apk add --no-cache git make
```
and run `make` [here](https://github.com/companieshouse/innovation-accounts-statistics-tool/pull/1/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R22)

The resulting image will be a tad heavier, but this would ensure consistency